lib/dndstrings.sty

```diff
@@ -1,5 +1,20 @@
 \ExplSyntaxOn % Use tilde (~) for spaces.
 
+% \addto\captions<language> cannot handle macros with parameters
+% We need to make our own parameters
+%\str_const:Nn \c__dnd_p1_str { |1| } % parameter 1
```

Comment:
It seems to be caused by the `1` inside `\c__dnd_p1_str`. This works:
```suggestion
\str_const:Nn \c__dnd_first_param_str { |1| } % parameter 1
```

Reply:
Oh, of course. I don't want the name to get too long, so that it can be used inline more easily, so I think I will steal a bit I saw in the babel docs and use roman numerals.

Reply:
In the end, even a short constant is really to long to use and makes it hard to read the captions in the file. It looks like enforcing a convention `|1|` is probably the way to go.